aws/tfstate-backend/Makefile

```diff
@@ -0,0 +1,7 @@
+## Apply the configuration
+apply:
```

Comment:
Rename `apply` to `init`.
`Apply` - confuse that this target can be used for update, but this target should be ran only once on `init`.

Reply:
Ok, that's fair.